components/nuki_lock/nuki_lock.cpp

```diff
@@ -158,6 +158,7 @@ void NukiLockComponent::control(const lock::LockCall &call) {
 
             if(this->lock_n_go_){
                 action = NukiLock::LockAction::LockNgo;
+                state = lock::LockState::LOCK_STATE_LOCKING;
```

Comment:
Does this make sense to you?
When Lock N Go is initiated the Nuki waits 30s before actually locking. In this period, I guess it can report that it's in the process of locking?